android/space/src/main/kotlin/net/twisterrob/inventory/android/space/ManageSpaceUiEventHandler.kt

```diff
@@ -0,0 +1,25 @@
+package net.twisterrob.inventory.android.space
+
+import net.twisterrob.inventory.android.arch.UiEventHandler
+import net.twisterrob.inventory.android.space.databinding.ManageSpaceActivityBinding
+import javax.inject.Inject
+
+internal class ManageSpaceUiEventHandler @Inject constructor(
```

Comment:
I like this, it's quite clean and helps getting rid of those instructions in the fragment/activity while still keeping them close to where they belong (binding).

Reply:
Yep, and since there's no Activity/Fragment in scope (unless injected), the possibilities are quite restricted to calling VM methods.